packages/lexical/src/LexicalSelection.ts

```diff
@@ -3084,7 +3084,7 @@ export function updateDOMSelection(
         : domSelection.rangeCount > 0
         ? domSelection.getRangeAt(0)
         : null;
-    if (selectionTarget !== null) {
+    if (selectionTarget instanceof HTMLElement) {
       // @ts-ignore Text nodes do have getBoundingClientRect
```

Comment:
While ignoring `scrollIntoView` will work I would be inclined to fix it as this logic is intended to run.

1) Range

```suggestion
    if (selectionTarget instanceof Text) {
      const range = document.createRange();
      range.select(selectionTarget);
      selectionRect = range.getBoundingClientRect();
```
2) HTMLElement wrapper assumption

We can assume that each Text is wrapped by an HTMLElement that consist is only one Text. I'm confident this should be true 100% of the cases in Lexical